### PR TITLE
Added new Device Types for AVM, Dell and Sophos

### PR DESCRIPTION
**Pull request by VentByte**
AVM:
- Fritz!Box 7510

Dell:
- PowerSwitch N3248P-ON
- PowerSwitch S4128F-ON

Sophos:
- AP6 420E
- APX320
- APX530
- XGS 126
- XGS 126

### CODE REVIEW

**Review by harryajc — CHANGES_REQUESTED** [review]

File: device-types/AVM/FRITZBox-7510.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    type: 1000base-t
+  - name: LAN2
+    label: LAN 2
+    type: 1000base-t
+  - name: LAN3
+    label: LAN 3
+    type: 1000base-t
+  - name: LAN4
+    label: LAN 4
+    type: 1000base-t
```
[the remark this answers is not in the document]
Comment (VentByte) [reply device-types/AVM/FRITZBox-7510.yaml]: I seem to have confused this with another model. It has been fixed with my latest changes

File: device-types/Dell/PowerSwitch-N3248P-ON.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    type: iec-60320-c14
+    maximum_draw: 1050
+    allocated_draw: 500
+interfaces:
+  - name: out-of-band
+    type: 1000base-t
+    mgmt_only: true
+  - name: Gi1/0/1
+    type: 1000base-t
+  - name: Gi1/0/2
```
[the remark this answers is not in the document]
Comment (VentByte) [reply device-types/Dell/PowerSwitch-N3248P-ON.yaml:28]: Done

File: device-types/Dell/PowerSwitch-S4128F-ON.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+console-ports:
+  - name: Console
+    type: usb-micro-b
+  - name: Serial
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+  - name: PS2
+    type: iec-60320-c14
```
[the remark this answers is not in the document]
Comment (VentByte) [reply device-types/Dell/PowerSwitch-S4128F-ON.yaml]: Done

**Review by harryajc — APPROVED** [review]

**Comment by harryajc** [comment]
Hi @VentByte

Thank you for making the requested changes, However there appears to be some merge conflicts with this PR. Please can you resolve these issues so that this PR can be merged

**Comment by VentByte** [comment]
Hi @harryajc
I've resolved the merge conflict, so the PR should now be ready to merge now.

**Review by harryajc — APPROVED** [review]

**Pull request merged**